types.go

```diff
@@ -42,11 +42,33 @@ type Options struct {
 	Output           io.Writer
 }
 
+// RESTClientGetter defines the options to customsize the RESTClient when
+// call  created
+type RESTClientOption func(*rest.Config)
+
+// The maximum length of time to wait before giving up on a server request
+// the created RESTClient will use DefaultTimeout: 32s
```

Comment:
```suggestion
// Timeout specifies the timeout for a RESTClient as a RESTClientOption.
// The default (if unspecified) is no timeout.
// See https://godoc.org/k8s.io/client-go/rest#RESTConfig.Timeout for more information.
```

@izturn Now I'm wondering - where did the `32 seconds` come from? :thinking:

Reply:
dig into the code, you will find the default value

Reply:
@izturn Can we agree on my above suggested change? The upstream documentation _should_ be the source of truth here, i suppose.

Reply:
i argee with you

Reply:
> @izturn Now I'm wondering - where did the `32 seconds` come from? thinking

I found it :
k8s.io/client-go@v0.24.3/discovery/discovery_client.go
```
// defaultTimeout is the maximum amount of time per request when no timeout has been set on a RESTClient.
// Defaults to 32s in order to have a distinguishable length of time, relative to other timeouts that exist.
defaultTimeout = 32 * time.Second
...
func setDiscoveryDefaults(config *restclient.Config) error {
	config.APIPath = ""
	config.GroupVersion = nil
	if config.Timeout == 0 {
		config.Timeout = defaultTimeout
	}
	...
	return nil
}
```

Reply:
Thanks @izturn, I've updated my comment accordingly.